[inductor] Disable remote fx graph cache in test_snode_runtime

masnesral opened the pull request:
Summary: Unfortunately we can't save / restore metrics.metrics.node_runtimes in the cache entries because these contain objects that don't pickle: `TypeError: cannot pickle 'PyCapsule' object`.

Test Plan: `buck2 test 'fbcode//mode/opt' fbcode//caffe2/test/inductor:snode_runtime -- --exact 'caffe2/test/inductor:snode_runtime - test_mm (caffe2.test.inductor.test_snode_runtime.ComputeBoundedTests)' --run-disabled --jobs 18 --stress-runs 10`

Differential Revision: D59705654


cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @peterbell10 @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @ColinPeppler @amjames @desertfire @chauhang

oulgen reviewed (changes requested):
hmm is this the right way to do this? if another file imports this file, then remote cache will also be disabled for them
i assume you need to disable local cache too?

masnesral commented:
@pytorchbot merge -i

masnesral commented:
@pytorchbot rebase

masnesral commented:
@pytorchbot rebase

masnesral commented:
@pytorchbot merge

masnesral committed:
[inductor] Disable remote fx graph cache in test_snode_runtime (#130655)

Summary:
Unfortunately we can't save / restore metrics.metrics.node_runtimes in the cache entries because these contain objects that don't pickle: `TypeError: cannot pickle 'PyCapsule' object`.

Pull Request resolved: https://github.com/pytorch/pytorch/pull/130655

Test Plan:
`buck2 test 'fbcode//mode/opt' fbcode//caffe2/test/inductor:snode_runtime -- --exact 'caffe2/test/inductor:snode_runtime - test_mm (caffe2.test.inductor.test_snode_runtime.ComputeBoundedTests)' --run-disabled --jobs 18 --stress-runs 10`

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx peterbell10 ipiszy yf225 chenyang78 kadeng muchulee8 ColinPeppler amjames desertfire chauhang

Reviewed By: oulgen

Differential Revision: D59705654

Pulled By: masnesral

masnesral commented:
@pytorchbot merge
